### PR TITLE
Add stepper component.

### PR DESCRIPTION
**Pull request by wwitzel3**
Signed-off-by: Wayne Witzel III <wayne@riotousliving.com>

image

image

### CODE REVIEW

**Comment by wwitzel3** [comment]
@GuessWhoSamFoo or @mklanjsek can either of you take a look at this and see if you can figure out what is causing the property error?

**Comment by mklanjsek** [comment]
Started looking at this right now

**Comment by wwitzel3** [comment]
Thanks @mklanjsek , the error I am seeing is in the issue, https://github.com/vmware-tanzu/octant/issues/1102#issuecomment-666341976

I only see it the first time I attempt to load the component in the Cnavas, this error does not occur when it first loads in to the docs.

**Comment by wwitzel3** [comment]
I'll rebase after review.

File: web/src/app/modules/shared/components/presentation/stepper/stepper.component.ts [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  steps: StepItem[];
+
+  constructor(
+    private formBuilder: FormBuilder,
+    private websocketService: WebsocketService
+  ) {
+    this.steps = [] as StepItem[];
+  }
+
+  ngOnChanges(changes: SimpleChanges): void {}
```
Comment (mklanjsek) [inline web/src/app/modules/shared/components/presentation/stepper/stepper.component.ts:64]: We should handle stepper definition changes and update the component.  Without that the `view` knob in Storybook will never work.
(if you feel the stepper definition will remain static, than maybe just remove the knob)

Comment (wwitzel3) [reply web/src/app/modules/shared/components/presentation/stepper/stepper.component.ts:64]: I'll remove it for now, we decided the initial implementation of the stepper would be static.

**Comment by wwitzel3** [comment]
@mklanjsek PTAL

**Pull request merged**